services/core-web/src/components/mine/Projects/MineProjectTable.js

```diff
@@ -29,7 +31,7 @@ const transformRowData = (projects) => {
       project_id: project.project_id || Strings.EMPTY_FIELD,
       project_name: project.project_title,
       proponent_project_id: project.proponent_project_id || Strings.EMPTY_FIELD,
-      project_contact: contact?.name || Strings.EMPTY_FIELD,
+      project_contact: name || Strings.EMPTY_FIELD,
```

Comment:
you can see in the snap below- but when it's assigned "" where the field is undefined above, it'll always end up defined as "" and the `EMPTY_FIELD` will never take effect. Could probably skip the defaulting with `|| ""`, and have project be assigned like firstName && last_Name && ${firstName} ${lastName} || EMPTY FIELD. Sometimes those fields want to stay undefined.

Reply:
This has been adjusted now in several places to use a structure along the lines of `[primaryContact?.first_name, primaryContact?.last_name].join(" ").trim();`. Thanks for the suggestion, it is a lot cleaner now I think.